src/integrationTest/resources/application.yml

```diff
@@ -16,3 +16,10 @@ security:
 
   authorisedServices:
     - "it"
+
+exui.role:
+  hmcts-admin: SuperUser
```

Comment:
Does this need to be called SuperUser? It may be confusing as we were previously calling the organisation_admin a superUser

Reply:
Yes. It got changed to prd-admin.